xbmc/platform/posix/filesystem/SMBDirectory.h

```diff
@@ -8,10 +8,13 @@
 
 #pragma once
 
-#include "MediaSource.h"
-#include "SMBFile.h"
```

Comment:
probably need to move this to `SMBDirectory.cpp`

Reply:
Have just dropped this for now whilst i work through the ins and outs of iwyu